### DISCUSSION TITLE
User Feedback Emails

### DISCUSSION THREAD

**Discussion by district6wc** [code4recovery/12-step-meeting-list#1147] [Q&A] [answered]
Has anyone been getting feedback on errors when trying to send an update with the "Request a change to this listing" function, I am getting "Could not initiate mail function" and a few members have reported the same?

**Comment by joshreisner** [comment, marked as answer]
Hi! This message comes from WordPress - it is telling you it's not able to send emails. See this topic for more info: https://github.com/code4recovery/12-step-meeting-list/discussions/1085

Setting up a SMTP plugin is probably your best next step, see @anchovie91471 's answer there

**Comment by district6wc** [comment]
Thank you for this info, I simply added the same email to the settings
panel again it seems to be working now.


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by joshreisner** [reply]
thanks for the followup Rose! so you had an email eg `feedback@intergroup.org` entered as an email address here:

<img width="448" alt="Screenshot 2023-06-17 at 10 58 03 AM" src="https://github.com/code4recovery/12-step-meeting-list/assets/1551689/00b42be0-616d-4ff2-825f-e67291c8e19d">

and then you clicked the x to remove and then re-entered the same exact address and it started working (without any other fixes)

**Reply by joshreisner** [reply]
sorry i should have put a question mark at the end there - does that sum up what happened for you? if so, that's new behavior. thanks

**Comment by district6wc** [comment]
Josh

Sorry for the delay my system crashed and have been on rebuild duty. But
yes, that seemed to be what happened at that point but back in it today and
I am getting the error again. Not sure how to manage this I am getting
other notices from WP for example when I make edits, so I am not sure the
SMTP is the issue.


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by joshreisner** [reply]
hm! i just filled out a form at https://alanonalateen6nc.org/meetings/ -- is that your website? looks like it went through, hopefully you received it. (i didn't get the error message)

my best theory is that by not setting up SMTP you are using your default hosting email transport, which may be throttled based on some kind of usage threshold. this would explain why sometimes it works and sometimes it doesn't - also this suggests that removing and re-adding the address might not have been the thing to actually fix the problem initially

if that's all true, then i think adding a SMTP might help. i wonder if your hosting provider can actually tell you where those emails are going, perhaps there is an error log they can check to see when emails are not sent?

i think we could start logging when the emails are not sent on our end, although it's not going to help us understand why it's happening unfortunately

**Comment by district6wc** [comment]
Josh
I appreciate the guidance here I am going to look into the SMTP. DO you
think resetting the google API would make a difference?
Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by tim-rohrer** [reply]
Hi Rose, 

Josh may chime in as well, but if you’re talking about the Google API configuration for maps, I can’t imagine that having any affect on your email issues.

Tim

**Reply by joshreisner** [reply]
Yes, that's right. 

Rose, one other option to consider is to switch to the TSML UI appearance. This pops open a simpler "mailto" button that triggers a new email that the user must send from their local phone / computer, rather than relying on WordPress to handle the emails

**Comment by district6wc** [comment]
The SMTP plugin seems to have resolved the issue, thank you for the tip. I
will also look into switching the UI.
Do you have anything to do with the PDF plugin called 12 Step Meeting PDF
Generator? I have some issues with functionality lately.


Thank you.

Yours in Service,
Rose Lanaro
Saturday Morning Live, Chapel Hill

District 6 Website Coordinator Panel 62
631-525-3328

**Reply by tim-rohrer** [reply]
We are not affiliated with 12 Step Meeting List Plugin Generator. If that isn’t working for you, we do have a [PDF generator service](https://github.com/code4recovery/pdf) you may want to try. 

@anchovie91471 is our lead maintainer on that now. Check out the link above and open questions there if you need more help.

-Tim
